math/lcm.go

```diff
@@ -0,0 +1,49 @@
+// Formula from https://en.wikipedia.org/wiki/Least_common_multiple
+// Find the least common multiple of two integers
+// by using the greatest common divisor.
+
+package main
+
+import (
+	"fmt"
```

Comment:
Consider using the already implemented `Gcd` algorithm, which you can find in the math directory.

```suggestion
	"fmt"
	"github.com/TheAlgorithms/Go/math/gcd"
```